app/modules/asset_groups/models.py

```diff
@@ -372,17 +372,40 @@ def getter(self):
     def get_custom_fields(self):
         return self.__class__.config_field_getter('customFields', default={})(self)
 
-    def get_encounters(self):
+    def _augment_encounter_json(self, encounter_data):
```

Comment:
refactored so that get_encounters and get_encounter use the same method, also populate much more to make it pretend to be an encounter